src/Makefile.am

```diff
@@ -115,13 +116,21 @@ GRIDCOIN_CORE_CPP = addrman.cpp \
 	script.cpp \
 	scrypt.cpp \
 	sync.cpp \
-        tally.cpp \
+    tally.cpp \
 	txdb-leveldb.cpp \
 	util.cpp \
 	version.cpp \
 	wallet.cpp \
 	walletdb.cpp
 
+GRIDCOIN_CONTRACT_H = \
```

Comment:
This separation, quite useless.

Reply:
I am planning too far ahead, I'll move it back GRIDCOIN_CORE_*